SConstruct

```diff
@@ -1482,25 +1482,36 @@ if pythonModuleEnv["PLATFORM"]=="darwin" :
 
 testEnv = env.Clone()
 
-testEnvLibPath = ":".join( testEnv["LIBPATH"] )
+testEnvLibPath = os.pathsep.join( testEnv["LIBPATH"] )
 if testEnv["TEST_LIBPATH"] != "" :
-	testEnvLibPath = testEnv["TEST_LIBPATH"] + ":" + testEnvLibPath
+	testEnvLibPath = testEnv["TEST_LIBPATH"] + os.pathsep + testEnvLibPath
 testEnvLibPath = testEnv.subst( testEnvLibPath )
 
-testEnv["ENV"][testEnv["TEST_LIBRARY_PATH_ENV_VAR"]] = testEnvLibPath
-if libraryPathEnvVar :
-	testEnv["ENV"][libraryPathEnvVar] = testEnvLibPath
-testEnv["ENV"]["IECORE_OP_PATHS"] = "test/IECore/ops"
+# Windows does not have a separate library path environment variable, so we need to use PATH.
+# But _replacing_ PATH would cause the compiler to be unavailable, so we append to PATH
+if testEnv["PLATFORM"] == "win32" : 
+	testEnv["ENV"][testEnv["TEST_LIBRARY_PATH_ENV_VAR"]] += testEnvLibPath
+else:
+	testEnv["ENV"][testEnv["TEST_LIBRARY_PATH_ENV_VAR"]] = testEnvLibPath
```

Comment:
fixed in https://github.com/ImageEngine/cortex/pull/1181/commits/a69ee40d9e7fea601c7ceb6d3c8917dc8252f5d3

Reply:
Fixed in https://github.com/ImageEngine/cortex/pull/1181/commits/cb6a247dd2d5bdc2fb2a650eb251aaebdef44cd4. 

I split this into its own commit because it was getting a little lost in the other SCons commit. We had been using `libraryPathEnvVar` directly in some modules instead of `env["TEST_LIBRARY_ENV_PATH"]` which meant Windows was avoiding replacing `PATH` since `libraryPathEnvVar` was `UNUSED_LIBRARY_PATH` regardless of what the options file specified for `TEST_LIBRARY_ENV_PATH`.

I standardized on `env["TEST_LIBRARY_ENV_PATH"]` throughout and used the pattern indicated in the commit comment to check that all of the dictionary entries are valid.